From: David Rosca <nowrep@gmail.com>
Date: Mon, 12 Dec 2016 10:15:40 +0100
Subject: Don't force logging with setFilterRules

Don't set the logging rules from inside the library, so the user
configuration (qtlogging.ini) is respected.

REVIEW: 129630
---
 src/manager.cpp | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/manager.cpp b/src/manager.cpp
index 6565299..c523e13 100644
--- a/src/manager.cpp
+++ b/src/manager.cpp
@@ -90,9 +90,6 @@ NetworkManager::NetworkManagerPrivate::NetworkManagerPrivate()
     , m_isWwanHardwareEnabled(false)
     , m_supportedInterfaceTypes(NetworkManager::Device::UnknownType)
 {
-    QLoggingCategory::setFilterRules(QStringLiteral("networkmanager-qt.debug = true"));
-    QLoggingCategory::setFilterRules(QStringLiteral("networkmanager-qt.warning = true"));
-
     connect(&iface, &OrgFreedesktopNetworkManagerInterface::DeviceAdded,
             this, &NetworkManagerPrivate::onDeviceAdded);
     connect(&iface, &OrgFreedesktopNetworkManagerInterface::DeviceRemoved,
